Dockerfile

```diff
@@ -1,3 +1,20 @@
-FROM botpress/server:v12_5_0
+FROM node:12-alpine
```

Comment:
Can you explain why you modified this Dockerfile and also added the new `build/module-builder/Dockerfile`?
In my mind, only 1 Dockerfile was necessary to build the Docker image to build modules?

Reply:
See the other discussion on this line:
> -> Added another variant of this Dockerfile idea under build/module-builder.
After issues have been resolved, only one new Dockerfile shall remain.

Approaches of these two are different, for now I have them both here so we can compare side by side. I think the more recent file is not using the right approach still. Git clone won't be the first step of the final image, but this allows me to explore which files are really needed for the build (in my first approach I made some wrong deductions), and can be reproduced easily